src/data/DBHelpers.hpp

```diff
@@ -107,6 +107,7 @@ struct NFTsData {
     ripple::AccountID owner;
     std::optional<ripple::Blob> uri;
     bool isBurned = false;
+    bool onlyUriChanged = false;  // Whether only the URI was changed
```

Comment:
why can't we just use the `owner` field? All other NFT transactions would the `owner` field